docs/support/tutorials/gis/eo_guide.md

```diff
@@ -0,0 +1,374 @@
+        * [Planet](https://www.planet.com/products/planet-imagery/)
+        * [WorldView](http://worldview3.digitalglobe.com/)
+        * [Proba-V](https://earth.esa.int/eogateway/catalog/proba-v-1km-333m-and-100m-products)
+        * ...
+    * RADAR (Radio Detection and Ranging) -> SAR (Synthetic Aperture Radar)
+        * [Copernicus Sentinel 1](https://sentinel.esa.int/web/sentinel/missions/sentinel-1)
+        * [Radarsat](https://www.asc-csa.gc.ca/eng/satellites/radarsat/default.asp)
+        * [TanDEM-X](https://www.dlr.de/content/en/missions/tandem-x.html)
+        * [TerraSAR-X](https://www.dlr.de/content/en/missions/terrasar-x.html)
+        * [ICEYE](https://www.iceye.com/)
+        * ...
+    * LiDAR (Light Detection and Ranging)
+        * [ICESat](https://icesat-2.gsfc.nasa.gov/)
+        * [GEDI](https://gedi.umd.edu/)
+        * ...
+    * Database of all current and future EO missions can be found in the [CEOS EO handbook database](http://database.eohandbook.com/database/missiontable.aspx)
+* Resolution
+    * Temporal: when and how often a certain area is visited
+    * Spatial: the area on the ground that each pixel covers
+    * Spectral: the area of the electromagnetic spectrum that is observed and spectral width of each band provided
+    * Radiometric: how many values are possible for each pixel (bit-depth)
+* Costs
+    * Free: e.g. Landsat, MODIS, Sentinel, ...
+    * Non free (but might be possible to get for free/less for research): e.g. WorldView, Spot, Planet, ...
+* Preprocessing needs
+    * Raw or pre-processed 
+* User experience and knowledge
+    * RADAR/LiDAR require solid background knowledge for processing and interpretation
+    * Optical  data is more easily interpreted and processed (and more pre-processed data is available)
+
+
+### Where do I find the data?
+
+The best place to get the data from depends on your needs: Do you want to download the data into your own processing environment or do you need a processing environment close to the data? The answer depends on what you want to do with the data and where it is located.
+
+Below is a (uncomplete) set of services, that provide download or download and processing (marked with *) capabilities:
+
+See also a list of other places on [CSC research pages](https://research.csc.fi/open-gis-data#intdata3).
+
+=== "CSC * "
+
+    * Puhti
+        * [List of all available datasets in Puhti](../../../data/datasets/spatial-data-in-csc-computing-env.md)
+        * Sentinel and Landsat mosaics of Finland provided by FMI and SYKE: ```/appl/data/geo/sentinel/s2```
+        * Every CSC user has **read** access to data stored on Puhti, no need to move it, unless you need to modify it
+    * Allas
+        * [List of all available geospatial datasets in Allas](../../../data/datasets/spatial-data-in-csc-computing-env.md)
+        * Sentinel-2 L2A data of crop growing Finland, growing seasons 2016-present, [usage instructions](https://a3s.fi/sentinel-readme/README.txt)
```

Comment:
Yea, this is a bit weird, but it actually is only those parts of Finland where crops are grown, and not crops growing in Finland. 
Maybe it could be `data of Finland where crops are grown` or `agriculturally active Finland` ? :D I don't know, they all sound weird.

Reply:
Oh, alright! Both of those options make more sense to me, but I don't know if it's just my general ignorance on the subject :D